### PR TITLE
Allow specifying log name at command line

### PR DESCRIPTION
**Pull request by rmunn**
A simple way to implement #223 without too many changes to the code.

### CODE REVIEW

File: Expecto/Expecto.fs [hunk 1]
```diff
@@ -1611,6 +1618,9 @@ module Tests =
     Global.initialiseIfDefault
       { Global.defaultConfig with
           getLogger = fun name -> LiterateConsoleTarget(name, config.verbosity, consoleSemaphore = Global.semaphore()) :> Logger }
+    match config.logName with
```
Comment (haf) [inline Expecto/Expecto.fs]: option.logName |> Option.iter setLogName

Comment (rmunn) [reply Expecto/Expecto.fs]: Done.

**Comment by AnthonyLloyd** [comment]
Thanks. I will hold this here until I get ok from downstream repos that they can release. Its breaking so should be a major version for all of us, will check.

**Comment by rmunn** [comment]
I would think that adding a command-line parameter would be the equivalent of adding a new API, a minor version number bump rather than a major bump (e.g., from 5.1 to 5.2 instead of to 6.0) since it's backwards-compatible (any build scripts that relied on the old API aren't affected since they don't use the new command-line parameter anyway).

Is there any reason why adding a new command-line parameter would be backwards-incompatible, i.e. would cause older build scripts to break?

**Comment by rmunn** [comment]
... Ah, but it's not *necessarily* backwards-compatible for anyone using the `ExpectoConfig` record type, because that type has grown an extra item. Anyone who uses `let config = { defaultConfig with verbosity = Something }` would still find it backwards-compatible at a source level, but not necessarily at a compiled-DLL level.

Yeah, if it *could* be non-backwards-compatible, them by SemVer rules, we need to bump the major version number.

**Comment by haf** [comment]
Better to bump major if it's not definitely backwards compatible and get people into that flow.

**Comment by AnthonyLloyd** [comment]
yep, its going to be 6.0.0

**Pull request merged**